demo/server/config-prod.json

```diff
@@ -28,8 +28,9 @@
   "redis": {
     "url": "redis://127.0.0.1:6379"
   },
-  "jwtSecret": "raincatcher",
-  "clientUrl": {
-    "portal": "http://localhost:9003/?url=http://localhost:8001"
+  "jwtSecret": "D837131FD17F62CB85FBD5919563086369691F4D42379C3596F811839A8992CBA1FBA88DF243BF2481940F112D339C33283BDFEF29A13612550EDAAAB7B5E061",
```

Comment:
ping @austincunningham  - that's update for demo passport auth setup. 
Users will need to change that in order to make sure that solution is secure.